scripts/guess-server-upgrade-base-image.sh

```diff
@@ -73,7 +73,7 @@ then
         print_vertica_k8s_img_with_tag $PRIVATE_REPO $PRIVATE_IMAGE "1f759615f0f723080b398edcf096a0bc8bc03aef-minimal"
     else
         # For all newer versions, we pick a random 24.3.0 image.
-        print_vertica_k8s_img_with_tag $PRIVATE_REPO $PRIVATE_IMAGE "20240701-minimal"
+        print_vertica_k8s_img_with_tag $PRIVATE_REPO $PRIVATE_IMAGE "20240712-minimal"
```

Comment:
You should change the version in tests/e2e-leg-9/multi-online-upgrade/30-initiate-upgrade.yaml and in `docker-push-extra-image`(in Makefile) to something newer than this version.